### ISSUE TITLE
supporting checker framework's stub files

### ISSUE THREAD

**Issue by chengniansun** [uber/NullAway#111]
Do you plan to support CF's stub files?  The stub file provides a mechanism to annotate library code. And there are already a bunch of stub files in CF.

**Comment by lazaroclapp** [comment]
@msridhar FYI, we also want to consume those stubs in the local version of the RxChecker, so we can just have a generic infrastructure for getting CF stubs into EP, assuming that doesn't exist already.

@chengniansun Any ideas on the later? Do you currently use CF stubs with EP checkers?

**Comment by Stephan202** [comment]
@lazaroclapp, I can't seem to fine more information about this "RxChecker" online. Its name makes it sound like an Error Prone plugin for reactive (anti-)patterns. If it is: is this tool open source or are there plans to make it so? :)

**Comment by lazaroclapp** [comment]
@Stephan202 The RxChecker (technically: the Rx Thread/Effect Checker) is an internal tool for the time being. It is more about threading issues in RxAndroid than any sort of general toolbox of Rx anti-patterns. There are plans to open source it, when and if we can, but no real timeline for that.

**Comment by Stephan202** [comment]
Thanks for clarifying!

**Comment by chengniansun** [comment]
I do not think we are using CF stub files in EP, and there seems to be no plan in the near future. 

But we do have a lot of stub files for CF.
